fastestimator/pipeline.py

```diff
@@ -363,7 +367,7 @@ def get_loader(self,
                                    output_keys,
                                    deep_remainder=False)
             # Results will be immediately converted to tensors, so don't need deep_remainder
-            batch_size = None if isinstance(data, BatchDataset) else batch_size
+            batch_size = None if data.fe_is_batched else batch_size
             data = DataLoader(op_dataset,
                               batch_size=batch_size,
                               shuffle=False if isinstance(data, BatchDataset) else shuffle,
```

Comment:
need to swap this is instance check to right? or maybe not due to the reset_index_maps thing? how will other datasets that are manually batching handle shuffle?

Reply:
likewise for the sampler